xbmc/filesystem/MusicDatabaseDirectory/DirectoryNode.cpp

```diff
@@ -46,6 +46,7 @@
 #include "utils/StringUtils.h"
 #include "guilib/LocalizeStrings.h"
 #include "music/MusicDbUrl.h"
+#include "settings/settings.h"
```

Comment:
missed an uppercase S here, should be settings/Settings.h and that's why it's failing on linux/android